sdk/core/src/data/types/buffer.rs

```diff
@@ -0,0 +1,27 @@
+use std::str;
+use multiversx_sc::api::ManagedTypeApi;
+use multiversx_sc::types::{ManagedBuffer, TokenIdentifier};
+use crate::data::types::native::NativeConvertible;
+
+impl<M: ManagedTypeApi> NativeConvertible for ManagedBuffer<M> {
+    type Native = String;
+
+    fn to_native(&self) -> Self::Native {
```

Comment:
There is some overlap in functionality between this and CodecFrom/CodecInto.

The difference is CodecFrom/CodecInto go many-to-many, whereas here it is many-to-one.

We used to have something similar in the past. Could be easier to use when returning data, since you only have 1 choice of conversion.

Something like this would rather belong in the codec or the base framework.

Reply:
The key advantage of having a many-to-one instead of a many-to-many regarding the output is that the end backend developer doesn't have to think about types. My motivation here is really to hide the "complexity" of what a ManagedType is.

Many-to-many will be useful to do a similar thing about parameter input though